Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -756,8 +756,10 @@ private enum ProductDropdownType {
                     }
                 }
 
+                int multiple = 0;
```

Comment:
I recommend to rename it to `numberOfEntries`. That would make it clearer what this variable is for.